jung-io/src/main/java/edu/uci/ics/jung/io/GraphMLWriter.java

```diff
@@ -13,6 +13,9 @@
 
 import com.google.common.graph.EndpointPair;
 import com.google.common.graph.Network;
+import edu.uci.ics.jung.io.graphml.AttributeType;
+
+import java.beans.XMLEncoder;
```

Comment:
@ziodave It looks to me that `java.beans.XMLEncoder` is never used, so this import should be removed if possible.

Reply:
done

Reply:
Ta!